assets/tasks/common.json

```diff
@@ -1648,6 +1563,8 @@
                 "TAG=arm64",
                 "GPU=${config:torizon_gpu}",
                 "docker-compose",
+                "-p",
+                "torizon",
```

Comment:
did you think that is better to pass `torizon` than leave it to create with the service name?

Reply:
This one is needed because `docker-compose down --remove-orphans` only stops containers with the same project name. So the debug container will not be stopped when trying to start the release container, and vice-versa, which generates problems like "This port is already allocated". 
This statement is also valid for trying to debug a new project after having debugged another one. This older one will not be stopped if it has a different project name.